page_layouts/product.html

```diff
@@ -0,0 +1,302 @@
+
+      </div>
+
+      <div class="gh-filter-wrap-col gh-filter-margin">
+        <div class="h5 text-bold gh-filter-title">Ratings</div>
+        <div class="gh-filter-wrap-row">
+          <input type="radio" name="rating" id="" value="4star-above">
+          <label for="" class="gh-filter-text">4 Star and above</label>
+        </div>
+        <div class="gh-filter-wrap-row">
+          <input type="radio" name="rating" id="" value="3star-above">
+          <label for="" class="gh-filter-text">3 Star and above</label>
+        </div>
+        <div class="gh-filter-wrap-row">
+          <input type="radio" name="rating" id="" value="2star-above">
+          <label for="" class="gh-filter-text">2 Star and above</label>
+        </div>
+
+      </div>
+
+      <div class="gh-filter-wrap-col gh-filter-margin">
+        <div class="h5 text-bold gh-filter-title">Sort by</div>
+        <div class="gh-filter-wrap-row">
+          <input type="radio" name="sort" id="" value="low-to-high">
+          <label for="" class="gh-filter-text">Price - Low to High</label>
+        </div>
+        <div class="gh-filter-wrap-row">
+          <input type="radio" name="sort" id="" value="high-to-low">
+          <label for="" class="gh-filter-text">Price - High to Low</label>
+        </div>
+        <div class="gh-filter-wrap-row">
+          <input type="radio" name="sort" id="" value="popularity">
+          <label for="" class="gh-filter-text">Popularity</label>
+        </div>
+
+      </div>
+
+    </aside>
+    <!-- filter ends-->
+
+    <!-- prod cards -->
+    <div class="gh-prod-wrap">
+
+      <!-- card start -->
+      <div class="gh-card card card-image shadow">
+        <!-- <div class="card-badge gh-green-bg">trending</div> -->
+        <div class="card-image-container">
+          <i class="ph-heart card-icon-heart gh-heart"></i>
+          <img src="https://imgcdn.floweraura.com/Express-Products-17-A.jpg" alt="image" class="card-image" />
+          <div class="card-rating">
+            <span class="rating">4.7</span>
+            <i class="fas fa-star card-icon-star"></i> | 78
+          </div>
+        </div>
+        <div class="card-content">
+          <div class="gh-card-title card-image-content-title">Indian Jade</div>
+          <div class="card-image-content-text">Set of 2 Plants</div>
+          <div class="gh-filter-wrap-row">
+            <span class="h6">Rs. 696</span> &nbsp;
+            <del class="h6">Rs. 839</del> &nbsp;
+            <span class="card-text-red">(17% OFF)</span>
+          </div>
+        </div>
+        <div class="card-btn-container gh-btn-container">
+          <button class="card-btn gh-btn gh-btn-primary">Buy Now</button>
+          <button class="card-btn gh-btn gh-btn-secondary">add to cart</button>
+        </div>
+      </div>
+      <!-- card end -->
+
+      <!-- card start -->
+      <div class="gh-card card card-image shadow">
+        <!-- <div class="card-badge gh-green-bg">trending</div> -->
```

Comment:
do remove unnecessary chunks of codes before merging

Reply:
done , as suggested.